# This service spawns the described models in the unreal
# world.

# Descriptions of the models to be spawned.
unreal_msgs/ModelDescription[] model_descriptions
---
# The returned arrays have the same size and the same index
# represents the same model.

# The returned instance_ids will contain the generated ids,
# if the id was set beforehand this will be equal to the
# instance_id in of the request.
unreal_msgs/InstanceId[] instance_ids

# true if model at index was spawned successfully.
bool[] success
